communication/best-practices.md

```diff
@@ -0,0 +1,82 @@
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your email account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications. All the steps below are basically for Gmail users.
+
+Note: If you are looking to create and manage lists as a chair or other organizer, 
+check this mailing list [guideline][mailinglist-guidelines] doc.
+
+- [Example for setting filters to Kubernets Mailing lists](#example-for-setting-filters-to-kubernets-mailing-lists)
+- [Examples for setting filters to Kubernetes Github notifications](#examples-for-setting-filters-to-kubernetes-github-notifications)
+                                                                                
+## Example for setting filters to Kubernetes Mailing lists           
+
+It depends on the SIG or/and WG you are involved in. 
+You would like to setup your filters in your Gmail account, so you don't need to have 
+all emails structured in one place.
+Create a filter following the procedure below:
+
+- In your Gmail account click on **Settings**:
+  - **Filters and Blocked Addresses** -> Scroll down and click **create a new filter**
+  - In the **to** fields write the email of the SIG's Google Group. 
+  - **Create filter** -> Check the box ** Apply the label** and create new
+label by choosing **New label...** in the dropdown list.
+  - Click on the **Create filter**.
+- You can also use filters directly for lists.
+  - E.g: Matches: list:"kubernetes-dev@googlegroups.com"
+  - Do this: Apply label "lists/kubernetes-dev"
+
+## Examples for setting filters to Kubernetes Github notifications
+
+In order to not get spammed by Github notifications and to get all the 
+notifications in the right place, here is an example of filters to create/apply
+on your Gmail account:
+
+- Stick a blue label on anything kubernetes-related:
+  - Matches: (kubernetes OR kubernetes-client OR kubernetes-sigs OR kubernetes-incubator
+OR kubernetes-csi)
+  - Do this: Apply label "k8s", Mark it as important
+- Archive your own actions (sending these is an option in Github's settings).  
+You can send them but also archive them, so whenever you need to see the history of 
+an issue you can:
+  - Matches: to:(your_activity@noreply.github.com) 
+    - Do this: Skip Inbox, Mark as read
+- Skip bot comments:
+  - Matches: (from:(notifications@github.com) (from:(k8s-merge-robot) OR 
+from:(Kubernetes Prow Robot) OR from:(k8s-ci-robot)))
+    - Do this: Skip Inbox, Mark as read
+- Skip push notifications:
+  - Matches: to:(push@noreply.github.com) 
+    - Do this: Skip Inbox, Mark as read 
+- Red label on things assigned to me and/or things request to be reviewed:
+  - Matches: to:(assign@noreply.github.com)
+    - Do this: Star it, Apply label "gh/p0", Mark it as important 
+  - Matches: to:(review_requested@noreply.github.com)
+    - Do this: Star it, Apply label "gh/rq", Mark it as important
```

Comment:
Can we have this as p1 or something? or maybe change the assign, etc labels to `gh/assign`? :thinking:

Reply:
I will change just the labels `gh/*`, it looks more good `gh/assign`